app/views/managers/show_request.html.erb

```diff
@@ -0,0 +1,33 @@
+<% @submissions.each do |submission| %>
+  <div class="container">
+    <% unless ["approved", "not approved", "reject"].include? submission[:status] %>
```

Comment:
instead of this check here in view you need to fetch all the submissions which are pending